views/marts/product/dim_excludable_servers.view.lkml

```diff
@@ -136,4 +136,12 @@ view: dim_excludable_servers {
     view_label: "Server: Exclusion Reasons"
   }
 
+  dimension: always_exclude {
+    type: yesno
+    sql:  ${has_reason_single_day_security_only} or ${has_reason_custom_build_version_format} or ${has_reason_single_day_server_side_telemetry_only} or ${has_reason_single_day_telemetry_only} or ${has_reason_ran_tests} or ${has_reason_single_day_user_telemetry_only} or ${has_reason_restricted_ip} or ${has_reason_invalid_server_id} or ${has_reason_internal_email} or ${has_reason_test_server} or ${has_reason_community};;
+    label: "Always Exclude"
+    description: "A combination of exclusion reasons which applies to production data analyses"
```

Comment:
The label and description are too vague. Let's find a more representative name for this group of exclusion reasons.